src/view/pages/BlogPost/components/Socials.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from 'view/components/common';
+
+enum Websites {
+    FACEBOOK = 'facebook',
+    GITHUB   = 'github',
```

Comment:
```suggestion
    //GITHUB   = 'github',
```

Then get rid of the default case in the switch statement; better to keep it exhaustive unless we have a good reason to handle a catch-all case